src/pykiso/tool/xray/cli.py

```diff
@@ -0,0 +1,70 @@
+import getpass
+
+import click
+
+from .xray import extract_test_results, upload_test_results
+
+
+@click.group()
+@click.option(
+    "--user",
+    help="Xray user id",
+    required=True,
+    default=None,
+    hide_input=True,
+)
+@click.option(
```

Comment:
You could add shortcut so user don't need to type `--password` everytime but could type `-p` for example, same for every option.